docker/Dockerfile_txsim

```diff
@@ -0,0 +1,48 @@
+# stage 1 Generate celestia-appd Binary
+FROM docker.io/golang:1.20.6-alpine3.17 as builder
+# hadolint ignore=DL3018
+RUN apk update && apk add --no-cache \
+    gcc \
+    git \
+    make \
+    musl-dev
+COPY . /celestia-app
+WORKDIR /celestia-app
+# we need the appd build as we might want to create an account
+# internally for txsimulation
```

Comment:
[question] I don't understand what this commend is trying to convey. Does "appd" refer to `celestia-appd`? If yes please fully specify.

What does this mean: "want to create an account internally"?

Reply:
The Docker image in its final form will need a 'master' celestia account to send txs. 
By internally, we mean here that the txsim image can: 

1. Create a keyring with celestia-appd
2. Do some logic on funding (this part I haven't implemented as there are too many ways on how to achieve this) 
3. Start the txsim binary

Reply:
Tracked in #2118